source/js/components/nsf-nav.js

```diff
@@ -32,9 +32,14 @@ var MenuLinks = React.createClass({
   },
   onScroll: function() {
     var links = this.state.links || [];
+    var hash = window.location.hash.replace(`#`, ``);
     var active = links[0] || ``;
     var scrollY = window.scrollY;
 
+    if (hash && !document.querySelector(`.nav-anchor#` + hash)) {
```

Comment:
Um we are using `id` wrong here for sure. I'm worried if we have a dev that's new to the codebase he/she will not know a special `classname` is needed for accordion & scroll hash to work properly. Anyways, since this project has tight deadline and if getting the accordion hash work has more priority then let's go ahead and merge this PR.